api/notd/store/retriever.py

```diff
@@ -118,3 +119,25 @@ async def get_collection_by_address(self, address: str, connection: Optional[Dat
             raise NotFoundException(message=f'Collection with registry {address} not found')
         collection = collection_from_row(row)
         return collection
+
+    async def get_collection_holding(self, address: str, ownerAddress: str, connection: Optional[DatabaseConnection] = None) -> List[Token]:
+        boughtTokens = []
+        soldTokens= []
+        query = TokenTransfersTable.select()\
+            .where(TokenTransfersTable.c.registryAddress == address)
```

Comment:
use brackets for multiline queries
```suggestion
        query = (
            TokenTransfersTable.select()
            .where(TokenTransfersTable.c.registryAddress == address)
        )
```